index.html

```diff
@@ -149,11 +149,11 @@ <h2 hidden>Game over</h2>
   // Display game over, cancel events
   function endGame(result) {
     if (result === "draw") {
-      gameOver.textContent = "Game over: Draw";
+      gameOver.textContent = "Game over, refresh to play again 🧑‍💻 🤖!: Draw";
     } else if (result === "human") {
-      gameOver.textContent = "Game over: You win";
+      gameOver.textContent = "Game over, refresh to play again 🧑‍💻 🤖!: You win";
```

Comment:
```suggestion
<h2 hidden>Game over! Want to play again?! Just click refresh. 🧑‍🚀!</h2>
```
Let's make it a bit more friendly. 🤓